admin/cordova_sandbox.sb

```diff
@@ -0,0 +1,106 @@
+(version 1) ; Fri Jan  6 12:01:53 2017
+
+; I M P O R T A N T:
+; /private/tmp/foobar below needs to be changed with the folder that holds the source code for Cordova project
```

Comment:
It's better to assume that current working directory should contain the source code and do not require any changes here?